lib/LedgerSMB/Template.pm

```diff
@@ -181,13 +190,16 @@ sub available_formats {
     local ($@);
     if (eval {require LedgerSMB::Template::LaTeX}){
         push @retval, 'PDF', 'PS';
-    }
+    } { warn $@ if $@; }
```

Comment:
hmm. why did you add the block around `warn` here? the above is equivalent to:
```perl
  }
  warn $@ if $@;
```

But, more importantly, we want to log it instead of warn, because warnings go to STDERR and loggers go everywhere our users want to, I'd say.